src/stores/middleware/errorHandler.ts

```diff
@@ -0,0 +1,14 @@
+import { Middleware, Dispatch } from 'redux';
+
+import { setError } from '../errors/errorsActions';
+
+export const errorHandler: Middleware = () => (
+  next: Dispatch,
+) => (action: any) => {
+  next(action);
+
+  if (action.type.includes('error') && !action.type.includes('reset')) {
```

Comment:
potentially you may have action called "@recipes/reset". To avoid issues in future - specify the exact action name that should be ignored